EpikCord/__init__.py

```diff
@@ -610,6 +718,17 @@ async def fetch_messages(
         after: Optional[str] = None,
         limit: Optional[int] = None,
     ) -> List[Message]:
+        """Fetches a list of messages from the parent channel or DM
+
+        Args:
+            around (Optional[str], optional): Get messages around this message ID (before and after the message was sent). Defaults to None.
+            before (Optional[str], optional): Get messages before this message ID. Defaults to None.
+            after (Optional[str], optional): Get messages after this message ID. Defaults to None.
+            limit (Optional[int], optional): Number of messages to return. Defaults to 50.
```

Comment:
Wtf `(Optional[...], optional)`. I think that `Optional[str]` is already clear enough lol

Reply:
Yeah...

Reply:
... I got the template from the VSCode extension Sphinx itself recommends https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring

Reply:
Remove it